src/main/java/controllers/ArticleController.java

```diff
@@ -17,18 +16,17 @@
     public void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, java.io.IOException {
 
-        ArticleBean articleBean = new ArticleBean();
+        ArticleService articles = new ArticleService();
```

Comment:
You should use interface instead of class.

Reply:
Teisingai